frontends/qt/.gitignore

```diff
@@ -2,3 +2,4 @@
 /config/
 /server/libserver.h
 /server/libserver.so
+/server/libbreez_sdk_bindings.so
```

Comment:
I also have `/server/libbreez_sdk_bindings.dylib` that could maybe be ignored?

Reply:
Looks like it's still in the Makefile. I added it too